apps/web/src/server/views/applications/components/folder/folder-delete.njk

```diff
@@ -0,0 +1,17 @@
+{% extends "../../layouts/folder-edit-property.layout.njk" %}
+
+{% block pageHeading %}
+	<p class="govuk-body-l govuk-!-margin-0 colour--secondary">{{ folderName }}</p>
+	<h1 class="govuk-label-wrapper">
+		<label class="govuk-label govuk-label--l">
```

Comment:
You can probably remove the label inside the H1

Reply:
cool, will add the change in the e2e test PR coming next so you don't have to reapprove!